apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/conf/dynamic/ConfigurationDiscoveryService.java

```diff
@@ -214,15 +213,12 @@ public WatcherHolder get(String name) {
         @Override
         public String toString() {
             StringBuilder registerTableDescription = new StringBuilder();
-            registerTableDescription.append("Following dynamic config property are available.").append(LINE_SEPARATOR);
-            registerTableDescription.append("---------------------------------------------").append(LINE_SEPARATOR);
             register.forEach((key, holder) -> {
                 AgentConfigChangeWatcher watcher = holder.getWatcher();
                 registerTableDescription.append("key:")
                                         .append(key)
-                                        .append("    value(current):")
-                                        .append(watcher.value())
-                                        .append(LINE_SEPARATOR);
+                                        .append("value(current):")
+                                        .append(watcher.value());
```

Comment:
I think you forget a `.` or `,` for the end?

Reply:
done.